src/lib.rs

```diff
@@ -284,19 +286,19 @@ impl CommandExt for process::Command {
             child.stdout.take();
             child.stderr.take();
 
-            StatusAsync2 {
-                inner: child,
-            }
+            StatusAsync2 { inner: child }
```

Comment:
I personally prefer always having a struct member initialization on its own line because it allows for cleaner diffs and edits when adding additional fields.

Reply:
This is configurable apparently: https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#struct_lit_single_line